Mathlib/Geometry/Manifold/IsManifold/InteriorBoundary.lean

```diff
@@ -270,21 +268,25 @@ lemma interiorPoint_inl (x : M) (hx : I.IsInteriorPoint x) :
 lemma boundaryPoint_inl (x : M) (hx : I.IsBoundaryPoint x) :
     I.IsBoundaryPoint (.inl x: M ⊕ M') := by
   rw [I.isBoundaryPoint_iff, extChartAt, ChartedSpace.sum_chartAt_inl]
-  dsimp
+  dsimp only [PartialHomeomorph.extend.eq_1, PartialEquiv.coe_trans, toPartialEquiv_coe,
```

Comment:
Back when landing this, Kim indicated that they preferred an un-squeezed `dsimp`. (I had specifically asked.)
Did you have a particular reason for squeezing, other than "non-terminal dsimps could be brittle"?

This is not something I'll insist on, but I'd like it to be a conscious decision.

The same applies to the other three `dsimp`s below.